public/index.html

```diff
@@ -153,7 +154,7 @@
     let code_el = document.getElementById('code');
     let formatted_el = document.getElementById('formatted');
     code_el.onkeyup = code_el.onchange = function() {
-        let parser = module.exports;
+        let parser = window['Codesplain_parse_' + lang_el.value];
```

Comment:
This should be `window['CodesplainParser']`, per the change made in https://github.com/maryvilledev/codesplain/commit/c2ca09c1f5b9dc5702e351e04c41db872d598c8e.